ffi/build.py

```diff
@@ -213,7 +213,7 @@ def main_posix(kind, library_ext):
 def main():
     if sys.platform == 'win32':
         main_windows()
-    elif sys.platform.startswith('linux'):
+    elif sys.platform.startswith(('linux', 'gnu')):
```

Comment:
If I encountered this whilst reading the code of `build.py` outside of the context of this PR, I'd not know what `'gnu'` means here (apologies for the harsh-sounding comment, but unfortunately it is the case that GNU Hurd has a relatively small userbase and there isn't broad awareness of the platform). Could you add a comment that notes what's going on here, e.g.:

```suggestion
    elif sys.platform.startswith(('linux', 'gnu')):
        # Linux and GNU Hurd both use the Linux Makefile
```

Reply:
makes sense, done